docs/source/morphpy.rst

```diff
@@ -74,6 +74,10 @@ General Parameters
 
 save: str or path
     Save the morphed function to a the file passed to save. Use '-' for stdout.
+get_diff: bool

```

Comment:
I wonder if we want to do either/or here.   A clearer logic would be that diff is saved if this is true, not saved if it is false, and also the case for the morph function.  So TT would save both.  To keep the API cleaner, we could have defaults of TF for morph and diff (the current behavior).